src/blocks/content-helper/class-content-helper.php

```diff
@@ -39,6 +49,12 @@ public function run(): void {
 			array(),
 			$content_helper_asset['version']
 		);
+
+		// Set wpParselyContentHelperPrefix for the "Post Stats" button links
+		// to point to the correct location.
+		$prefix                = trailingslashit( 'https://dash.parsely.com/' . esc_js( $this->parsely->get_api_key() ) ) . 'find';
+		$analytics_link_prefix = 'window.wpParselyContentHelperPrefix = "' . $prefix . '";';
+		wp_add_inline_script( 'wp-parsely-block-content-helper', $analytics_link_prefix, 'before' );
```

Comment:
Summary of what's happening here:

1. Here, we inject a browser global containing the Parse.ly API prefix.
2. Client-side, we [fetch posts data](https://github.com/Parsely/wp-parsely/blob/develop/src/blocks/content-helper/content-helper-provider.ts#L64) from our REST API endpoint.
3. We process that data and [mix-in a `scriptUrl` property](https://github.com/Parsely/wp-parsely/blob/develop/src/blocks/content-helper/content-helper-provider.ts#L80), which uses this browser global It is the only reference in the code to this global.

Instead of this, I'd recommend just defining the correct `scriptUrl` property server-side and adding it the API response here:

https://github.com/Parsely/wp-parsely/blob/557d0ddbebe9bbff4dd2d5aba33fb12568163efa/src/Endpoints/class-analytics-posts-api-proxy.php#L46

Then you can remove the `processData` function and with it the need for this global.

Reply:
Let me know if what I've done is close to what you were thinking.